lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +22,7 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
```

Comment:
@douglashall If this is a site configuration, where would the `DISPLAY_TOS_IN_MODAL_ON_REGISTRATION_PAGE` setting be defined? I tried on my devstack's `lms.envs.private` module but I think that doesn't work.